src/main/java/com/gmail/val59000mc/scenarios/scenariolisteners/RandomizedBlockDropsListener.java

```diff
@@ -0,0 +1,63 @@
+package com.gmail.val59000mc.scenarios.scenariolisteners;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.inventory.ItemStack;
+
+import com.gmail.val59000mc.scenarios.ScenarioListener;
+import com.gmail.val59000mc.utils.RandomUtils;
+
+public class RandomizedBlockDropsListener extends ScenarioListener{
+	
+	private List<Material> materialitems;
+	private HashMap<Material, ItemStack> dropList; 
+	
+	public RandomizedBlockDropsListener() {
+		
+		materialitems = new ArrayList<>();
+		dropList = new HashMap<Material, ItemStack>();
+	}
+
+	@Override
+	public void onEnable() {
+		//Create new arraylist of materials that are all items
+	for(int i = 0; i< Material.values().length; i++) {
+		if(Material.values()[i].isItem()) {
+			 materialitems.add(Material.values()[i]);
+			}
+		}
+	}
```

Comment:
Fix the spacing of the code.
Change "fori" loop to a "foreach" loop.
`for(Material material : Material.values()){}`